src/animalShelter/AnimalShelter.ts

```diff
@@ -71,16 +74,16 @@ export default class AnimalShelter {
         );
         return;
       }
-      if (animal.getType() === "Crocodile") {
+      if (animal instanceof Crocodile) {
```

Comment:
is it possible to write the code using switch instead of multiple if-s ?

Reply:
This will also be addressed in a future task